pkgs/tools/misc/ntfy-sh/default.nix

```diff
@@ -15,8 +21,21 @@ buildGoModule rec {
 
   doCheck = false;
 
+  nativeBuildInputs = with pkgs; [
+    debianutils
+    mkdocs
+    nodejs
+    python3
+    python3Packages.mkdocs-material
+    python3Packages.mkdocs-minify
+  ];
+
   preBuild = ''
-    make cli-deps-static-sites
+    sed -i 's# /bin/echo# echo#' Makefile
+    sed -i 's#react-scripts build#DISABLE_ESLINT_PLUGIN=true react-scripts build#' web/package.json
```

Comment:
I think this can be replaced with just setting `DISABLE_ESLINT_PLUGIN=true` when running `make`.

Reply:
Thanks! I don't know how I missed that...